conf/apollo.config

```
@@ -0,0 +1,35 @@
+
+params {
+    config_profile_name = 'apollo'
+    config_profile_description = 'COH Apollo cluster profile provided by nf-core/configs.'
+    config_profile_contact = 'Denis O\'Meally (@drejom) or HPRCC Helpdesk (HPRCC_Help@coh.org)'
+    config_profile_url = 'https://www.cityofhope.org/ Intranet http://hprcc.coh.org/'
+    max_memory = 1500.GB
+    max_cpus = 72
+    max_time = 500.h
+    igenomes_base = '/ref_genome/igenomes'
+}
+
+process {
+    executor = 'slurm'
+    queue = 'all'
+    beforeScript = 'module load singularity'
+}
+
+singularity {
+    enabled = true
+    cacheDir = '/opt/singularity-images/nextflow'
```

```suggestion
    libraryDir = '/opt/singularity-images/nextflow'
```

Setting `cacheDir` would tell nextflow to store images there, which I guess should not be possible for most users?